__tests__/api.test.js

```diff
@@ -70,9 +69,130 @@ describe('GET - /api/articles/:article_id', () => {
         );
       });
   });
-})
+
+  test('GET comment count- ', () => {

```

Comment:
this isn't the valid endpoint for getting the articles comments. You should be making a GET request to /api/articles/:article_id/comments. You have done so for the POST request but not for the get request.